### ISSUE TITLE
[BUG] Wrong data type

### ISSUE THREAD

**Issue by OgreTransporter** [AcademySoftwareFoundation/OpenImageIO#4210]
In the commit 22ef9b2887f13ffab1109573925182aeaf0e4f1, the function `std::string Filesystem::unique_path(string_view model)` under Windows uses `std::wstring` as the data type for the variables `modelStr` and `name`. The library cannot be built because the function `exists` cannot be called in this way!
https://github.com/AcademySoftwareFoundation/OpenImageIO/blob/dd9eb9b6b836d5e2967c222158c865ecaf7b9266/src/libutil/filesystem.cpp#L526

Compiler error MSVC:
```
filesystem.cpp(526,14): error C2664: "bool OpenImageIO_v2_6_1::Filesystem::exists(OpenImageIO_v2_6_1::string_view) noexcept" : 
 Conversion of argument 1 from "std::wstring" to "OpenImageIO_v2_6_1::string_view" not possible
```

**Comment by lgritz** [comment]
Sorry about that. Working on a fix right now.

**Comment by lgritz** [comment]
Fixed proposed in #4211. Are you able to pull down that patch and try it out? If so, please let us know here.

**Comment by OgreTransporter** [comment]
I use the same fix. Works!
